src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -36,4 +36,8 @@
     <include file="changes/202402021110-insert-wishlist-dev.xml" relativeToChangelogFile="true" context="@dev"/>
     <include file="changes/202402081000-wishlist-item-table.xml" relativeToChangelogFile="true"/>
     <include file="changes/202402081010-insert-wishlist-item-table-dev.xml" relativeToChangelogFile="true" context="@dev"/>
+    <include file="changes/202402031338-token-table.xml" relativeToChangelogFile="true"/>
```

Comment:
Have you got `202402031338-token-table.xml` file?

Reply:
Of course there is, and my colleague Igor forgot to include it in this file.